README.md

```diff
@@ -11,7 +11,7 @@ on: [push]
 jobs:
   test:
     steps:
-      - uses: datadog/agent-github-action@v1
+      - uses: datadog/agent-github-action@v0.1.0
```

Comment:
@cg14823 can we use `v0` instead? So we don't have to keep updating the README with every minor/patch revision

Reply:
If you don't put the exact version it doesn't work. https://github.com/DataDog/ci-webhooks-test/runs/4881448001?check_suite_focus=true